tests/test_ext.py

```diff
@@ -40,6 +40,9 @@
     "ngettext.html": '{{ ngettext("%(num)s apple", "%(num)s apples", apples) }}',
     "ngettext_long.html": "{% trans num=apples %}{{ num }} apple{% pluralize %}"
     "{{ num }} apples{% endtrans %}",
+    "pgettext.html": '{{ pgettext("company", "Apple") }}',
+    "npgettext.html": '{{ npgettext("company", "%(num)s apple", "%(num)s apples",'
```

Comment:
I know it's just an example, but I think using `"fruit"` as the context would be much more logical here. Also because one would usually not mark Apple-the-company for i18n.